### PR TITLE
Add warning that this repo is going to be archived

### PR DESCRIPTION
**Pull request by cookel2**
### What

Updated the readme to show that the repo is being archived. This is required as per this ticket:
https://trello.com/c/HvW6RbMt/1312-remove-unneeded-services-from-sandbox-staging-and-live

### How to review

Check that the warning in the readme makes sense.

### Who can review

Anyone but me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
+
 ---
 
 :warning: This repository will be archived in November 2022 as it is no longer in development. :warning:
```
Comment (ghost) [inline README.md:4]: Just wondering if you intended the :warning: to be bold or italic such as ***warning*** instead?

Comment (ghost) [reply README.md:4]: Ah interesting so 'warning' surrounded by a pair of ':'  gives a nice little 'attention' icon. I don't see that in my ide!

**Review by ghost — COMMENTED** [review]
Just a general comment - I'm getting errors on the table in the README. I know you didn't change it but my IDE gave me a nice little 'fix' option which re-formatted it properly. Do you have that?

**Comment by cookel2** [comment]
Thanks for the review Patrick. Unfortunately GoLand does not complain about the table and does not change the table if I run its formatter either.

**Review by ghost — APPROVED** [review]

**Pull request merged**